service/dap/server.go

```diff
@@ -273,11 +273,9 @@ const (
 	maxStringLenInCallRetVars = 1 << 10 // 1024
 )
 
-var (
-	// Max number of goroutines that we will return.
-	// This is a var for testing
-	maxGoroutines = 1 << 10
-)
+// Max number of goroutines that we will return.
```

Comment:
Avoid unrelated reformats, they pollute blames and make merge conflicts unnecessarily harder.

Reply:
My editor auto-formats using `gofumpt`, so it formatted some extra editted files. Will revert such changes.